LibraBFT/Impl/Consensus/RoundManager.agda

```diff
@@ -1,82 +1,32 @@
 {- Byzantine Fault Tolerant Consensus Verification in Agda, version 0.9.
 
-   Copyright (c) 2020, 2021, Oracle and/or its affiliates.
```

Comment:
We need to keep the years from the beginning of the file's life in the copyright notices, which is from 2020 in this case.

Reply:
The `Impl` `RoundManager` is new code this year.
The `ImplFake` `RoundManager`. has `2020, 2021`
So I assume no change is required here?

Reply:
Ah got it, thanks.